tests/cpydiff/module_array_constructor.py

```diff
@@ -0,0 +1,11 @@
+"""
+categories: Modules,array
+description: overflow checking is not implemented
+cause: micropython implements implicit truncation in order to reduce code size and execution time
+workaround: mask value
```

Comment:
I suggest: "if CPython compatibility is needed then mask the value explicitly"

Reply:
I have updated the PR.